.circleci/config.yml

```
@@ -152,13 +152,11 @@ commands:
               docker push "quay.io/rhacs-eng/apollo-ci:${TAG}" && break || sleep 15
             done
 
-            if [[ "<< parameters.image-flavor >>" == "stackrox-build" || "<< parameters.image-flavor >>" == "scanner-build" ]]; then
-              for i in {1..5}; do
-                docker login -u "$QUAY_STACKROX_IO_RW_USERNAME" -p "$QUAY_STACKROX_IO_RW_PASSWORD" quay.io
-                docker tag "${IMAGE}" "quay.io/stackrox-io/apollo-ci:${TAG}"
-                docker push "quay.io/stackrox-io/apollo-ci:${TAG}" && break || sleep 15
-              done
-            fi
+            for i in {1..5}; do
```

```suggestion
            for _ in {1..5}; do
```
This is a nit pick. i is not used so _ should be better.